ci/steps.yml

```diff
@@ -175,13 +175,33 @@ steps:
     displayName: Build accelbubble example application to test for android
 
   ##----------------------------------------------------
-  # determine Windows build system
+  # Cache Powershell Modules in $MODULES_FOLDER
+  - task: Cache@2
+    inputs:
+      key: 'pwsh modules | "$(Agent.OS)" | Pscx | VSSetup'
```

Comment:
This key will need to change in the future if we need to download other modules, or if these modules are updated. If we don't change the key, Azure will just load the same cached copies over and over again.

It would probably make sense to add the version numbers for the Pscx and VSSetup modules to this key, but I don't know how to do that.

If we can safely assume that we won't ever need updated versions of these modules, maybe this implementation is OK.

Reply:
When Microsoft releases new version of Visual Studio such as VS2022,  and we want to test with it for sample project, we may want to update these modules to recognize new VS version.

(cf. https://github.com/Pscx/Pscx/pull/89)

Reply:
That's a good point. 

One reasonable option would be to lock the key to the `pool.vmImage` property, which is `windows-2019`. When we move up to a new windows image that includes VS2022, we would cache new versions of the modules.